libs/langchain/langchain/llms/bedrock.py

```diff
@@ -267,7 +267,12 @@ def _prepare_input_and_invoke(
             text = LLMInputOutputAdapter.prepare_output(provider, response)
 
         except Exception as e:
-            raise ValueError(f"Error raised by bedrock service: {e}")
```

Comment:
since `e` is part of the output, wouldn't you see anyways that it was a ThrottlingException? what does original error message look like?

Reply:
This was the error

```diff
except Exception as e:
            raise ValueError(f"Error raised by bedrock service: {e}")
```

Reply:
This was the actual error

```diff
        except Exception as e:
            raise ValueError(f"Error raised by bedrock service: {e}")
```

since e is part of the output and it was a ThrottlingException

Reply:
Just want to make the error more understandable